cmake/modules/SetUpMacOS.cmake

```diff
@@ -7,6 +7,26 @@
 set(ROOT_ARCHITECTURE macosx)
 set(ROOT_PLATFORM macosx)
 
+# https://gitlab.kitware.com/cmake/cmake/issues/19222
+if(CMAKE_VERSION VERSION_LESS 3.14.4)
+  if(CMAKE_GENERATOR STREQUAL "Ninja")
+    find_program(NINJAPROG ninja DOC "looking for Ninja to be used")
+    if (NINJAPROG)
+      execute_process(COMMAND ${NINJAPROG} --version
+        OUTPUT_VARIABLE NINJAVERSION
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+        RESULT_VARIABLE NINJAVERSIONRES)
+      if(${NINJAVERSION} EQ 0)
```

Comment:
Probably my lack of cmake understanding but aren't those those saying
```
if ( ninja_version == 0 && ninja_version > 1.8.2) do stuff
```
i.e. it never triggers?

Reply:
Wrong var name, fixed!